src/aria-snapshots/button-test-aria-ts-button--button-disabled-false-.yml

```diff
@@ -0,0 +1 @@
+- button "Label"
```

Comment:
@noahgelmancs: Here's what the initial batch of ARIA snapshots looks like.

Reply:
Very helpful so far. They give us confidence when making changes, and we've [already](https://github.com/CrowdStrike/glide-core/pull/751/files#diff-a8d73aaeff0b6f6a27f99185dbae1b6a848246a63ff348e348f617216085060a) caught a few bugs.